src/App.jsx

```diff
@@ -4,8 +4,8 @@ import {
   Routes,
   Route,
 } from 'react-router-dom';
-import Book from './components/Book';
-import Books from './components/Books';
+import BooksList from './components/BooksList';
+import BooksLibrary from './components/BooksLibrary';
 import Categories from './components/Categories';
 import NavBar from './components/shared/NavBar';
 import './App.css';
```

Comment:
- [ ] [OPTIONAL] Your app is running with winnings, kindly fix them  so the app can run normally without errors or warnings, this is highly recommended 

![Screenshot_20220723_154833](https://user-images.githubusercontent.com/25227022/180610348-2c46c713-98d4-44b3-aae7-49eb4f7ce6ed.png)